nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/FetchSFTP.java

```diff
@@ -81,6 +81,10 @@
         properties.add(SFTPTransfer.STRICT_HOST_KEY_CHECKING);
         properties.add(SFTPTransfer.USE_COMPRESSION);
         properties.add(SFTPTransfer.PROXY_CONFIGURATION_SERVICE);
+        properties.add(SFTPTransfer.CIPHERS_ALLOWED);
```

Comment:
I would change the order of properties so these new ones are not in the middle of the proxy-related props. Arguably, the new ones could go at the end since many or most users will not be interested in changing the settings.